src/main/java/svc/data/citations/CitationDAO.java

```diff
@@ -93,15 +96,21 @@ public Citation mapRow(ResultSet rs, int i) {
 			try {	
 				citation.id = rs.getInt("id");
 				citation.citation_number = rs.getString("citation_number");
-				citation.citation_date = rs.getDate("citation_date");
+				citation.citation_date = (rs.getDate("citation_date")!=null)?rs.getDate("citation_date").toLocalDate():null;
```

Comment:
Seems inefficient to try to get the thing twice. Just get it in a local var and check. Also - "rs" (ResultSet) may have a "hasProperty" type of call.